x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -625,6 +626,41 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
           />
         </EuiFormRow>
       </EuiDescribedFormGroup>
+
+      {/* Include aliases */}
+      <EuiDescribedFormGroup
+        title={
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.snapshotRestore.restoreForm.stepLogistics.includeAliasesTitle"
+                defaultMessage="Include aliases"
+              />
+            </h3>
+          </EuiTitle>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.snapshotRestore.restoreForm.stepLogistics.includeAliasesDescription"
+            defaultMessage="Allow aliases to be restored along with associated indices."
+          />
+        }
+        fullWidth
+      >
+        <EuiFormRow hasEmptyLabelSpace={true} fullWidth>
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.snapshotRestore.restoreForm.stepLogistics.includeAliasesLabel"
+                defaultMessage="Include aliases"
```

Comment:
I'd sync this with the h3 above.
```suggestion
                defaultMessage="Restore aliases"
```